### DISCUSSION TITLE
Speed increase/decrease shortcut should also work in multiplayer rooms

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#30671] [Ideas] [answered]
Would be especially useful in daily challenge, but it would be nice if it could work in any multiplayer room (if the option is available)
Thanks!

**Comment by Joehuu** [comment, marked as answer]
This is already tracked as an issue in https://github.com/ppy/osu/issues/29491.

**Reply by vatei** [reply]
Woops I forgot to check issues too. Thanks
